openshift-ci/build-root/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM registry.ci.openshift.org/openshift/release:golang-1.20
+FROM registry.ci.openshift.org/openshift/release:golang-1.21
 
 RUN yum -y install dnf httpd-tools
```

Comment:
If not required, you can remove this. `yum` seems to be the problem here

Reply:
@jgwest - can you remove the yum command as pointed by @deepsm007 ? It worked for us for gitops-operator https://github.com/redhat-developer/gitops-operator/pull/743